incubator/hnc/internal/validators/anchor.go

```diff
@@ -79,8 +79,13 @@ func (v *Anchor) handle(req *anchorRequest) admission.Response {
 	switch req.op {
 	case k8sadm.Create:
 		// Can't create subnamespaces in excluded namespaces
-		if config.EX[pnm] {
-			msg := fmt.Sprintf("The namespace %s is not allowed to create subnamespaces. Please create subnamespaces in a different namespace.", pnm)
+		if config.ExcludedNamespaces[pnm] {
+			msg := fmt.Sprintf("The namespace %q is not allowed to create subnamespaces. Please create subnamespaces in a different namespace.", pnm)
```

Comment:
I know this isn't a new message for this PR, but... the namespace %q isn't creating the namespace, the user is :) Also the second sentence doesn't really add a lot, but it's more important now that the list is configurable to give users an idea of what the problem is.

Your new error messages below are really good ("Cannot set the excluded namespace %q as a child of another namespace") so maybe follow that pattern here?

Reply:
Updated all three here.